testsuite/cluster-test/src/main.rs

```diff
@@ -144,41 +146,73 @@ fn setup_log() {
 
 struct ClusterUtil {
     cluster: Cluster,
-    aws: Aws,
-    prometheus: Prometheus,
+    aws: Option<Aws>,
+    prometheus: Option<Prometheus>,
 }
 
 struct ClusterTestRunner {
     logs: LogTail,
     cluster: Cluster,
     health_check_runner: HealthCheckRunner,
-    deployment_manager: DeploymentManager,
+    deployment_manager: Option<DeploymentManager>,
```

Comment:
Similarly, whatever methods ClusterTestRunner has, should require aws and deployment_manager, so we can keep this field as non-optional